src/solution_part2.kt

```diff
@@ -1,3 +1,13 @@
 fun main(){
- print("I love Kotlin!!!!")
+ val firstPlayerHealthPoints = 79
+ val secondPlayerHealthPoints = 45
+
+ if (firstPlayerHealthPoints in 51..90) println("the player's health is fine")
+ else if (firstPlayerHealthPoints < 50) println("the player's health is awful")
+
+ when(secondPlayerHealthPoints){
+  in 90..100 -> println("the player's health is fine")
+  in 0..50 -> println("the player's health is awful")

```

Comment:
logic error 
val firstPlayerHealthPoints = 79
  val secondPlayerHealthPoints = 45

 fun main(){
  val PlayerHealthPoints = 79
 when(PlayerHealthPoints){
    in 50..89 -> println("the player's health is fine")
    in 0..49 -> println("the player's health is awful")
   else -> println("Score not valid")
  }